packages/@sfcc-bff/basketapi/src/api/models/Basket.ts

```diff
@@ -62,6 +65,15 @@ class Basket {
             price: apiBasket?.orderPriceAdjustments?.[0].price ?? 0.0,
         };
 
+        this.shippingLevelPriceAdjustment = {
```

Comment:
Added this for showing the shipping level discount in the order totals on the client side.